data_utils/raw_dataset_loader.py

```diff
@@ -30,17 +30,35 @@ def load_raw_data(self, dataset_path: str):
         dataset_path: str
             Directory path where raw files are stored.
         """
+
+        atom_feature_dim = config["atom_featurs"]["dim"]
+        atom_feature_loc = config["atom_featurs"]["locations"]
+        prop_dim = config["properties"]["dim"]
+        prop_loc = config["properties"]["locations"]
+        LSMS = True
```

Comment:
I did not use "LSMS" because there are some specific operations, e.g., line 130, associated with the material datasets we have. Not sure if that exists in all LSMS.